### DISCUSSION TITLE
Table driven scenario

### DISCUSSION THREAD

**Discussion by bansal63** [getgauge/gauge#1848] [General]
Hi,

We are working on using this experimental.
https://docs.gauge.org/writing-specifications.html?os=linux&language=javascript&ide=vscode#table-driven-scenario

Are there any timelines on when reporting plugin will accommodate this feature?

On trying this, we see that scenario pass count is not populated correctly. 
Is there a way we can set this pass count manually at the end of step? 
Or is there any other alternative of reporting?

**Comment by zabil** [comment]
> Are there any timelines on when reporting plugin will accommodate this feature?

There are no plans for this at the moment. Any pull request to fix this will be merged.

> Or is there any other alternative of reporting?

You can try using the [xml-report](https://github.com/getgauge/xml-report) plugin to parse the result.

**Reply by sschulz92** [reply]
May I ask if  there are any plans in this year maybe? :)
